src/support/services/ticket_service.py

```diff
@@ -0,0 +1,24 @@
+from django.contrib.auth.models import User
+
+from support.models import Ticket
+
+
+class TicketService:
+    @staticmethod
+    def find_support() -> User:
+        """Find staff-user who have minimum tickets"""
+        all_staff = User.objects.filter(is_staff=True)
+        staff_dict = {}
+        for staff in all_staff:
```

Comment:
[N+1 issue](https://adamj.eu/tech/2020/09/01/django-and-the-n-plus-one-queries-problem/)

you can create a single request to db to retrieve a user with less tickets. Use select_related, annotate, Count